src/App.test.js

```diff
@@ -7,3 +7,48 @@ it('renders without crashing', () => {
   ReactDOM.render(<App />, div);
   ReactDOM.unmountComponentAtNode(div);
 });
+
+
+const puppeteer = require('puppeteer');
+
+describe('Products title', () => {
+  test('Title loads correctly', async () => {
```

Comment:
Nice) Now you can split it into 2 test cases.
Hint: You can move all preparation staff into some helper

Reply:
Done. Will improve.